python/py-flatbuffers/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-flatbuffers
+# version should be the same as devel/flatbuffers
+version             1.12.0
+license             Apache-2
+platforms           darwin linux
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+description         Python runtime library for use with the Flatbuffers \
+                    serialization format.
+long_description    {*}${description}
+
+homepage            https://google.github.io/flatbuffers/
+
+python.versions     38
+
+if {${name} ne ${subport}} {
+
+    depends_build-append \
+                    port:flatbuffers \
```

Comment:
I see that flatbuffers installs a static library. Does py-flatbuffers use it? If so, is there a way to make flatbuffers install a dynamic library (either in addition or instead)? If so, then this flatbuffers build dependency would change to a library dependency.

Reply:
This is the case of the Python source being included in the same tarball as the `flatbuffers` source itself.

If you look in the Github repo for flatbuffers: https://github.com/google/flatbuffers

... you can see a sub-directory for a number of different programming languages containing the language-specific libraries, of which Python is one.

As far as I can tell, the Python source does not use the dylib.  It looks to be pure Python.

I tried to make `py-flatbuffers` a sub-port within the `flatbuffers` port, which would eliminate the issue of having to keep version numbers in sync, as well as having to get the source out of a totally different port.

Unfortunately that didn't work out so well as the `python` portgroup is already doing sub-port shenanigans for the different Python versions.  Maybe I'm doing it wrong.

Reply:
The `flatbuffers` Python code as far as I can tell looks to be pure-Python, and does not look to be using the dylib

Reply:
> Unfortunately that didn't work out so well as the `python` portgroup is already doing sub-port shenanigans for the different Python versions. Maybe I'm doing it wrong.

No you're correct. The python portgroup likes to be by itself. This is the same reason why there are separate libvirt and py-libvirt portfiles with comments at the top reminding me to keep them in sync.